roles/common/defaults/main.yml

```diff
@@ -48,3 +48,7 @@ nagios_allowed_hosts: "127.0.0.1"
 nrpe_selinux_packages:
   - libsemanage-python
   - policycoreutils-python
+
+nrpe_selinux_fedora_packages:
```

Comment:
These should get defined as `nrpe_selinux_packages` in `roles/common/vars/fedora_31.yml`.